src/main/java/net/shadowmage/ancientwarfare/npc/config/AWNPCStatics.java

```diff
@@ -46,7 +46,7 @@
 	private Configuration foodConfig;
 	private static final String foodSettings = "01_food_settings";
 	private HashMap<String, Integer> foodValues;
-	private int foodMultiplier = 750;
+	private int foodMultiplier;

```

Comment:
This should really be turned into private static int same as all the other config values in Statics classes